microcosm_flask/tests/swagger/test_definitions.py

```diff
@@ -30,7 +30,6 @@
 def test_build_swagger():
     graph = create_object_graph(name="example", testing=True)
     ns = Namespace(
-        path="/v1",
```

Comment:
This was another bug; in the namespace logic, the version input is used to build the actual endpoint path.

(This is still confusing and deserves another round of cleanup and some unit tests.)